@here/harp-geojson-datasource/lib/GeoJsonTile.ts

```diff
@@ -201,7 +202,7 @@ export class GeoJsonTile extends Tile {
             path,
             this.getRenderStyle(technique),
             this.getLayoutStyle(technique),
-            priority,
+            getPropertyValue(priority, this.mapView.zoomLevel),
```

Comment:
I'm not sure if everything should be ``MaybeInterpolatedProperty``... Some cases like this look a bit weird.

I'd stick to stylable properties being interpolated, and be a bit more strict with ones that modify scene/runtime logic (like this).

Just an opinion though, would be nice to have more input on this.

Reply:
I'm simply fixing already working feature - we have interpolated priority prop in our styles right now, but not on every technique. And our code supports interpolated properties on almost all techniques. So I think, make all techniques to support priority interpolation would be good decision.